src/Metadata/Resource/Factory/OperationResourceMetadataFactory.php

```diff
@@ -46,7 +46,7 @@ public function create(string $resourceClass): ResourceMetadata
 
         if (null === $resourceMetadata->getItemOperations()) {
             $resourceMetadata = $resourceMetadata->withItemOperations($this->createOperations(
-                $isAbstract ? ['GET', 'DELETE'] : ['GET', 'PUT', 'DELETE']
+                $isAbstract ? ['GET', 'DELETE'] : ['GET', 'PUT', 'DELETE', 'PATCH']
```

Comment:
I suggest to enable the PATCH operation only if JSONAPI support is enabled. WDYT?